browser.js

```diff
@@ -77,6 +77,10 @@ function registerShortcuts(username) {
 			$('button[data-testid="dmComposerSendButton"]').click();
 		}
 	});
+
+	Mousetrap.bindGlobal('backspace', () => {
```

Comment:
Shouldn't be global? This will make backspace in textarea's go back instead of delete a character.